src/mapfile2js/parse/resolveSymbolset.ts

```diff
@@ -35,11 +35,12 @@ export function resolveSymbolset(mapfile: Mapfile): Mapfile {
   // fallback to mapserver defaults if not specified
   if (!symbolsetPath) {
     symbolsetPath = `${__dirname}/../../../data/mapfiles/symbols.sym`;
+  } else if (!fs.existsSync(symbolsetPath)) {
+    console.error(`Non existent symbolset path: ${symbolsetPath}`);
+    symbolsetPath = `${__dirname}/../../../data/mapfiles/symbols.sym`;
```

Comment:
You define the same path 3 lines above, please, use a `const`

Reply:
Done!